.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: required
+sudo: false
```

Comment:
ok. keeping outdated config rules seems like an odd preference since the current Travis environment is specifically designed to not need this rule, but I guess it's not a blocker.